src/capi/azext_capi/custom.py

```diff
@@ -293,6 +294,18 @@ def update_management_cluster(cmd, yes=False):
         raise UnclassifiedUserFault("Couldn't upgrade management cluster") from err
 
 
+def create_resource_group(cmd, rg_name, location, yes=False):
```

Comment:
where is this being used?

Reply:
Will be used on creating an AKS management cluster. I did not do the change in this PR as will address in other issue about AKS management cluster

Reply:
Got it. I usually try to avoid checking in "dead" / unused code. It might make sense to add this function and its test in the AKS PR once it's actually being used and keep this PR focused on handling the error for RG not found.

Reply:
Got it, yeah, this PR shifted a lot will do the correction and move the method and test to other PR